test/e2e/storage/external/testdata/example.yaml

```diff
@@ -1,5 +1,8 @@
 StorageClass:
   FromExistingClassName: example
+# TODO - for reviewers - SnapshotClass isn't in the example, should I remove this?
```

Comment:
/cc @xing-yang @jsafrane

Reply:
I think that depends on whether it is used by any tests.

Reply:
This file is an example for external storage tests (i.e. it isn't used directly in any k/k test, it only serves as documentation by example)

Reply:
Left it in for now, because this is just an example we can always remove it later